server/game/cards/12-2T2D/OldWashoeClub.js

```diff
@@ -0,0 +1,52 @@
+const DeedCard = require('../../deedcard.js');
+const GameActions = require('../../GameActions/index.js');
+
+class OldWashoeClub extends DeedCard {
+    setupCardAbilities(ability) {
+        this.action({
+            title: 'Controller Shootout, Boot: Send an opposing A-value dude home booted.',
```

Comment:
I would use shorter titles. Player should know the card they are using, so it should just help him to be sure what ability he is triggering.
For example: 
`Shootout: Send A-dude home booted`
`Shootout: Discard a Sidekick`

Reply:
Done